sdk/lib/src/imagekitio-angular/ik-upload/ik-upload.component.ts

```diff
@@ -45,6 +46,13 @@ export class IkUploadComponent implements AfterViewInit {
     this.buttonRef && this.buttonRef.addEventListener('click', ()=>{this.el.nativeElement.children[0].click()});
   }
 
+  abort() {
+    if (this.xhr) {
+      this.xhr.abort();
+      console.log('Upload aborted');
```

Comment:
remove console log added during development/debugging.

Reply:
done removed the log in [8887b1a](https://github.com/imagekit-developer/imagekit-angular/pull/92/commits/8887b1a64d7edbc8e8fb4fccb37417dce7656d75)